src/Cake.Svn/SvnAliases.Info.cs

```diff
@@ -0,0 +1,255 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using Cake.Core;
+using Cake.Core.Annotations;
+using Cake.Core.IO;
+using Cake.Svn.Info;
+
+namespace Cake.Svn
+{
+    public static partial class SvnAliases
+    {
+        /// <summary>
+        /// Check if the <paramref name="directoryPath"/> is controlled by Subversion.
```

Comment:
IMHO `is a subversion wokring copy` would be clearer than `is controlled by Subversion`